app/src/ui/about/about.tsx

```diff
@@ -5,14 +5,16 @@ import { Row } from '../lib/row'
 import { Button } from '../lib/button'
 import { ButtonGroup } from '../lib/button-group'
 import { Dialog, DialogError, DialogContent, DialogFooter } from '../dialog'
-import { Octicon, OcticonSymbol } from '../octicons'
 import { LinkButton } from '../lib/link-button'
 import { updateStore, IUpdateState, UpdateStatus } from '../lib/update-store'
 import { Disposable } from 'event-kit'
 import { Loading } from '../lib/loading'
 import { RelativeTime } from '../relative-time'
 import { assertNever } from '../../lib/fatal-error'
 import { ReleaseNotesUri } from '../lib/releases'
+import { encodePathAsUrl } from '../../lib/path'
+
+const DesktopLogo = encodePathAsUrl(__dirname, 'static/logo-64x64@2.png')
```

Comment:
Just a minor addition to convey that it's the retina version. Thanks!

```suggestion
const DesktopLogo = encodePathAsUrl(__dirname, 'static/logo-64x64@2x.png')
```